charts/k3s/templates/pdb.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if and (.Values.enableHA) (gt (int .Values.replicas) 1) -}}
```

Comment:
Would you mind adding a helm value that would allow users to opt out of PDB creation, please?

Reply:
Implemented in 8da6d5ab3e905143794a7cb6fd5aece9937ca596

Reply:
Looks good! Thank you.